bindings/java/iota-client-java/src/main/java/org/iota/types/Milestone.java

```diff
@@ -0,0 +1,15 @@
+package org.iota.types;
+
+import com.google.gson.JsonObject;
+
+public class Milestone extends AbstractObject {
+
+    public Milestone(JsonObject jsonObject) {
+        super(jsonObject);
+    }
+
+    public Milestone(String jsonObject) {
+        super(jsonObject);
+    }
+
+}
```

Comment:
For many types this is the same apart from the actual name, is there no less repetitive way with less code to do the same?

Reply:
Not that I'm familar with; there are no marcos like in Rust
Using abstract classes or interfaces to avoid repetitive way is the only way imo